website/blog/2023-06-14-install-ferretdb-mongodb-alternative-ubuntu.mdx

```diff
@@ -0,0 +1,321 @@
+Restart=always
+RestartSec=2
+TimeoutStopSec=5
+SyslogIdentifier=ferretdb
+
+[Install]
+WantedBy=multi-user.target
+```
+
+In the above command, remember to replace the credentials for **PostgreSQL** before you proceed with the below command to reload the system daemon:
+
+```sh
+sudo systemctl daemon-reload
+```
+
+Start and enable the service:
+
+```sh
+sudo systemctl enable ferretdb
+sudo systemctl start ferretdb
+```
+
+Check the status of the service:
+
+```sh
+$ systemctl status ferretdb
+●  ferretdb.service - FerretDB
+     Loaded: loaded (/etc/systemd/system/ferretdb.service; enabled; vendor preset: enabled)
+     Active: active (running) since Fri 2023-04-14 18:35:07 EAT; 5s ago
+   Main PID: 9424 (ferretdb)
+      Tasks: 7 (limit: 4629)
+     Memory: 3.9M
+        CPU: 10ms
+     CGroup: /system.slice/ferretdb.service
+             └─9424 ferretdb --mode=normal --listen-addr=127.0.0.1:27017 "--postgresql-url=postgres://ferretuser:Passw0rd!@127.0.0.1:5432/ferretuser"
+
+Apr 14 18:35:07 tutornix.lab systemd[1]: Started FerretDB.
+Apr 14 18:35:07 tutornix.lab ferretdb[9424]: 2023-04-14T18:35:07.544+0300        INFO        ferretdb/main.go:231        Starting FerretDB v1.0.0...        {"version": "v1.0.0", "commit": ">
+Apr 14 18:35:07 tutornix.lab ferretdb[9424]: 2023-04-14T18:35:07.545+0300        INFO        listener        clientconn/listener.go:95        Listening on TCP 127.0.0.1:27017 ...
+Apr 14 18:35:07 tutornix.lab ferretdb[9424]: 2023-04-14T18:35:07.545+0300        INFO        listener        clientconn/listener.go:183        Waiting for all connections to stop...
+Apr 14 18:35:07 tutornix.lab ferretdb[9424]: 2023-04-14T18:35:07.546+0300        INFO        debug        debug/debug.go:95        Starting debug server on http://127.0.0.1:8088/
+```
+
+## Step 4 – Use FerretDB on Ubuntu
+
+Once installed and started, FerretDB can be used just like MongoDB.
+You can connect to it using the Mongo shell.
+
+You need to have the Mongo Shell installed on your system before you proceed.
+First, add the MongoDB repo to your system.
+
+```sh
+sudo apt install wget curl gnupg2 software-properties-common apt-transport-https ca-certificates lsb-release -y
+curl -fsSL https://www.mongodb.org/static/pgp/server-6.0.asc|sudo gpg --dearmor -o /etc/apt/trusted.gpg.d/mongodb-6.gpg
+echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu $(lsb_release -cs)/mongodb-org/6.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-6.0.list
+```
+
+Install the Mongo Shell on Ubuntu:
+
+```sh
+sudo apt update && sudo apt install mongodb-mongosh
+```
+
+Now connect the FerretDB:
+
+```sh
+mongosh
+```
+
+Sample Output:
+
+![Run FerretDB on Ubuntu](/img/blog/ferretdb-ubuntu/run-ferretdb-ubuntu.webp)
+
+Once connected, switch to the database created on PostgreSQL
+
+```sh
+test> use ferretuser
+switched to db ferretuser
+ferretuser> show dbs
+public  0 B
```

Comment:
I'm not sure why we run `show dbs` there. Normally, one would run a similar command to show that database was created. But `use` command does not actually create the FerretDB/MongoDB database. And that's a real confusion point – just check our community Slack